frontend/pages/policies/PolicyPage/PolicyPage.tsx

```diff
@@ -114,17 +114,12 @@ const PolicyPage = ({
   }, [isRouteOk, teamIdForApi, policyTeamId, setPolicyTeamId]);
 
   useEffect(() => {
```

Comment:
Do we want to revert the changes to this file?

Reply:
I reverted the change to the use effect dependencies; the only changes I now made is combining two useEffects that were run on initial mount only

Reply:
I'm not sure they can be combined like that. My understanding is that first one is meant to run on mount while the other one runs a cleanup function on dismount. (It is not very clear so we should probably look into an alternative to the pattern in the medium-term, but I'd be inclined to spec that as a tech debt story.)

Reply:
Ok I just reverted it